tracer/src/Datadog.Trace/Iast/IastRequestContext.cs

```diff
@@ -70,16 +70,22 @@ internal void AddVulnerability(Vulnerability vulnerability)
         }
     }
 
-    internal void AddRequestBody(object body, object? bodyExtracted)
+    internal void AddRequestBody(object? body, object? bodyExtracted)
     {
         try
         {
+            _executedTelemetryHelper?.AddExecutedSource(IastInstrumentedSources.RequestBody);
+
```

Comment:
Should this be here? 🤔 If `body` and `bodyExtracted` are both `null`, you don't do anything, so I don't know if you should increment `AddExecutedSource(IastInstrumentedSources.RequestBody);`? 🤔

Reply:
I think that it should. We are measuring the executed instrumentation points and if we have reached this point is because we have triggered an instrumentation point, even if there a re no body values to taint.